kornia/augmentation/random_generator/_2d/resize.py

```diff
@@ -87,4 +87,4 @@ def forward(self, batch_shape: torch.Size, same_on_batch: bool = False) -> Dict[
 
         _input_size = torch.tensor(input_size, device=_device, dtype=torch.long).expand(batch_size, -1)
 
-        return dict(src=src, dst=dst, input_size=_input_size, resize_to=torch.as_tensor(self.resize_to))
+        return dict(src=src, dst=dst, input_size=_input_size, output_size=torch.as_tensor(self.resize_to))
```

Comment:
```suggestion
        return dict(src=src, dst=dst, input_size=_input_size, output_size=torch.as_tensor(self.output_size))
```
maybe not for now but makes it more clear too to read